docs/source/tutorials/index.rst

```diff
@@ -1,11 +1,13 @@
 .. _tutorials:
    Tutorials
- 
+
 .. toctree::
    :maxdepth: 2
 
    compile
 
+   shell_shortcuts
```

Comment:
I am thinking about moving the shell shortcuts to other tutorial entry. Do you have a plan what other tutorials may be added in this category? Otherwise, I may move it to  ``design_flows``

Reply:
Yes I am planning to append this page with some small tutorials. Especially to get started quickly with after installation or docker pull. You can decide where to place it, but it will be more of a tutorial.

Reply:
I agree that its content fits what a tutorial need. A better way may be along with the installation. Should we create an entity about environment setup, e.g., ``Tool Setup``, where we place the compilation guidelines, docker pull, and this tutorial, as subchapters?

Reply:
I have similar information on this page, do you want to add some more and create separate section ?
https://openfpga--224.org.readthedocs.build/en/224/tutorials/compile/

Reply:
I mean that the how-to-compile section will be a subsection of a section 'Tool Setup' (or find a better name).
Under this section, we have
- how-to-compile
- Tool support
- Shell shortcut